README.md

```diff
@@ -33,9 +46,15 @@ make deps
 make run
 ```
 
-## Local (Development)
+> You can use the `make watch` command to run the application on watching mode, allowing it to be restarted automatically when the code changes.
+
+## Docker installation and usage
+
+### Requirements
+
+*   Install docker engine (<https://docs.docker.com/install/>)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)